src/sql/connection.py

```diff
@@ -7,6 +7,7 @@
 from IPython.core.error import UsageError
 import difflib
 
+IS_SQLALCHEMY_ONE = int(sqlalchemy.__version__.split(".")[0]) < 2
```

Comment:
let's be more explicit

```suggestion
IS_SQLALCHEMY_ONE = int(sqlalchemy.__version__.split(".")[0]) == 1
```